tools/src/main/java/io/github/ygojson/tools/dataprovider/impl/yugipedia/mapper/CardTable2Mapper.java

```diff
@@ -0,0 +1,79 @@
+package io.github.ygojson.tools.dataprovider.impl.yugipedia.mapper;
+
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+import io.github.ygojson.tools.dataprovider.impl.yugipedia.model.CardTable2;
+
+@Mapper(uses = MarkupStringMapper.class)
+public abstract class CardTable2Mapper {
+
+	public static final CardTable2Mapper INSTANCE = Mappers.getMapper(
+		CardTable2Mapper.class
+	);
+
+	/**
+	 * Pattern for the CardTable2 content
+	 */
+	private static final Pattern CARDTABLE2_CONTENT_PATTERN = Pattern.compile(
+		"(?s).*\\{\\{CardTable2(.*)\n}}",
+		Pattern.DOTALL
+	);
+
+	/**
+	 * Group on the {@link #CARDTABLE2_CONTENT_PATTERN} where the content is.
+	 */
+	private static final int CARDTABLE2_CONTENT_GROUP = 1;
+
+	/**
+	 * Split pattern fore each fiel on the CardTable2 content.
+	 */
+	private static final Pattern CARDTABLE2_FIELD_SPLIT_PATTERN = Pattern.compile(
+		"\n\\| "
+	);
+
+	/**
+	 * Split pattern fore each key=value pair on the CardTable2 content.
+	 */
+	private static final Pattern CARDTABLE2_KEY_VALUE_SPLIT_PATTERN =
+		Pattern.compile(" += ");
+
+	public CardTable2 wikitextToCardTable2(final String wikitext) {
```

Comment:
Change the method to, as we would like to be consistent with all mapper methods names:

```suggestion
	public CardTable2 mapWikitextToCardTable2(final String wikitext) {
```